src/nodedef.cpp

```diff
@@ -586,6 +588,7 @@ void ContentFeatures::deSerialize(std::istream &is)
 
 	try {
 		node_dig_prediction = deSerializeString(is);
+		leveled_max = readU8(is);
```

Comment:
`readU8` unfortunately doesn't throw an exception so this doesn't work
The following should work (not sure if function name is correct):
```c++
    u8 tmp = readU8(is);
    if (is.is_eof())
        throw SerializationError();
    leveled_max = tmp;
```